test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -240,37 +221,19 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           `window.ethereum.request(${switchEthereumChainRequest})`,
         );
 
-        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
-        await driver.findElement({
-          text: 'Use your enabled networks',
-          tag: 'p',
-        });
-
         await driver.switchToWindowWithUrl(DAPP_ONE_URL);
 
         await driver.clickElement('#sendButton');
 
         await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
 
-        await driver.clickElement({ text: 'Cancel', tag: 'button' });
```

Comment:
this needs to stay the same as well since we are testing the scenario when there is an approval for permitting a chain

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/27980/commits/7f7646f7a81ee1c998cfddde8a2185b9099fa80c